libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -2257,6 +2257,11 @@ void AP_AHRS::getCorrectedDeltaVelocityNED(Vector3f& ret, float& dt) const
     ret.z += GRAVITY_MSS*dt;
 }
 
+void AP_AHRS::set_failure_inconsistent_message(const char *estimator, const char *axis, float diff_rad, char *failure_msg, const uint8_t failure_msg_len) const
```

Comment:
This is my patch, but I could've used `snprintf_failure_inconsistent_message` here